contracts/src/AccountRegistry.sol

```diff
@@ -903,6 +900,28 @@ contract AccountRegistry is Initializable, EIP712Upgradeable, Ownable2StepUpgrad
         emit RecoveryAddressUpdated(accountIndex, oldRecoveryAddress, newRecoveryAddress);
     }
 
+    ////////////////////////////////////////////////////////////
+    //                      Owner Functions                   //
+    ////////////////////////////////////////////////////////////
+
+    /**
+     * @dev Sets the validity window for historic roots. 0 means roots never expire.
+     */
+    function setRootValidityWindow(uint256 newWindow) external virtual onlyOwner onlyProxy onlyInitialized {
+        uint256 old = rootValidityWindow;
+        rootValidityWindow = newWindow;
+        emit RootValidityWindowUpdated(old, newWindow);
+    }
+
+    /**
+     * @dev Set an updated maximum number of authenticators allowed.
+     */
+    function setMaxAuthenticators(uint256 newMaxAuthenticators) external onlyOwner onlyProxy onlyInitialized {
+        uint256 old = maxAuthenticators;
+        maxAuthenticators = newMaxAuthenticators;
+        emit MaxAuthenticatorsUpdated(old, maxAuthenticators);
```

Comment:
do we need to enforce a max value on maxAuthenticators since we use are using a bitmap